pkg/resources/network_rule_state_upgraders.go

```diff
@@ -0,0 +1,16 @@
+package resources
+
+import (
+	"context"
+)
+
+func v0_94_1_NetworkRuleStateUpgrader(ctx context.Context, rawState map[string]any, meta any) (map[string]any, error) {
```

Comment:
\+ theoretically the state upgrader could be generic, because field names are the same.
But still, I'm not sure if it's necessary to have state upgrader.

Reply:
Removed.